platform/src/components/aws/agent-runtime.ts

```diff
@@ -0,0 +1,863 @@
+                      );
+                    });
+
+                    return env;
+                  },
+                ),
+                authorizerConfiguration: args.authorizerConfiguration
+                  ? output(args.authorizerConfiguration).apply((auth) => ({
+                      customJwtAuthorizer: auth.customJwtAuthorizer
+                        ? {
+                            discoveryUrl: output(
+                              auth.customJwtAuthorizer.discoveryUrl,
+                            ),
+                            allowedAudience:
+                              auth.customJwtAuthorizer.allowedAudience,
+                            allowedClients:
+                              auth.customJwtAuthorizer.allowedClients,
+                          }
+                        : undefined,
+                    }))
+                  : undefined,
+              },
+              {
+                parent,
+                provider: awsNativeProvider,
+                dependsOn: [roleDelay, image],
+              },
+            ),
+          );
+        },
+      );
+    }
+
+    function registerReceiver() {
+      parent.registerOutputs({
+        _hint: all([parent._workloadIdentityArn]).apply(
+          ([workloadIdentityArn]) => {
+            return workloadIdentityArn ? workloadIdentityArn : undefined;
+          },
+        ),
+        _receiver: {
+          links: output(args.link || [])
+            .apply(Link.build)
+            .apply((links) => links.map((link) => link.name)),
+          environment: output(args.environment || {}),
+        },
+      });
+    }
+  }
+
+  /**
+   * The ARN of the agent runtime.
+   */
+  public get agentRuntimeArn() {
+    if (!this.agentRuntime) return output(undefined);
+    return this.agentRuntime.agentRuntimeArn;
+  }
+
+  /**
+   * The ID of the agent runtime.
+   */
+  public get agentRuntimeId() {
+    if (!this.agentRuntime) return output(undefined);
+    return this.agentRuntime.agentRuntimeId;
+  }
+
+  /**
+   * The version of the agent runtime.
+   */
+  public get agentRuntimeVersion() {
+    if (!this.agentRuntime) return output(undefined);
+    return this.agentRuntime.agentRuntimeVersion;
+  }
+
+  /**
+   * The ARN of the workload identity for the agent runtime.
+   */
+  public get workloadIdentityArn() {
+    return this._workloadIdentityArn;
+  }
+
+  /**
+   * The underlying [resources](/docs/components/#nodes) this component creates.
+   */
+  public get nodes() {
+    return {
+      /**
+       * The AWS Bedrock Agent Core Runtime resource.
+       */
+      agentRuntime: this.agentRuntime,
+      /**
+       * The IAM role used by the agent runtime.
+       */
+      role: this.role,
+    };
+  }
+
+  /** @internal */
+  public getSSTLink() {
+    return {
+      properties: {
+        arn: this.agentRuntimeArn,
+        id: this.agentRuntimeId,
+        workloadIdentityArn: this.workloadIdentityArn,
+      },
+    };
+  }
```

Comment:
if i'm not mistaken, the permissions for invoking the agent runtime should be included here similar to the `Bucket` component: 

https://github.com/sst/sst/blob/324df1fe7dc78bed6f228afc2867a1bff5a17407/platform/src/components/aws/bucket.ts#L1652-L1656

it would look something like this for your case:

```suggestion
  /** @internal */
  public getSSTLink() {
    return {
      properties: {
        arn: this.agentRuntimeArn,
        id: this.agentRuntimeId,
        workloadIdentityArn: this.workloadIdentityArn,
      },
      include: [
        permission({
          actions: ["bedrock:*"],
          resources: [this.agentRuntimeArn, interpolate`${this.agentRuntimeArn}/*`],
        }),
      ],
    };
  }
```

this way when you link the agent runtime to a lambda or service, it will have the necessary permissions to invoke it

Reply:
yes thanks for the comments I fixed it !